changelogs/unreleased/gh-9643-origin-in-priv-options.md

```diff
@@ -0,0 +1,6 @@
+## feature/box
```

Comment:
It is not easy to deduce a motivation behind this feature from the documentation request. You're saying the following.

> This option allows to differentiate privileges based on their origin. Moreover, if a privilege was granted using `box.schema.*.grant()` with an origin, it can only be revoked using `box.schema.*.revoke()` with the same origin. Also, now the same privilege with different origin can be applied.

Our motivation is to don't revoke a privilege granted by two actors if one actor revokes it. This simple idea can be expressed in one sentence, but it is the most important thing to say about the feature, I think.

Reply:
For now I left a simple commit-message. I'll rewrite the doc-request in a commit about `info()`.

Reply:
It was decided to not make this feature public, so doc-request in not needed.

Reply:
> motivation is to don't revoke a privilege granted by two actors if one actor revokes it

I think that it is important point and it makes things more clear, at least for me.

BTW, we can say it a bit better:

> motivation is to don't *actually* revoke a privilege granted by two actors if one actor revokes it

Reply:
Fixed.